### PR TITLE
Returning light

### PR DESCRIPTION
**Pull request by Haden2**
Didn't do the code to return the light when catching the enemy but I know how. My issue is I added a flare to the light to give it a source and to make it look nicer, but I cannot get the flare to turn back on when pressing Q

### CODE REVIEW

File: Assets/Scripts/ToggleFlashlight.cs [hunk 1]
```diff
@@ -4,16 +4,22 @@
...
     if (Input.GetKeyDown ("q")) 
 	{
-      if (light.intensity == 0) //If the light was off, turn it on.
-        light.intensity = lightIntensity;
+      if (light.intensity == 0 && lf.brightness == 0) //If the light was off, turn it on.
+       { 
+			light.intensity = lightIntensity; //.75
+			lf.brightness = lensBright; //2
+		}
       else
```
Comment (thomas-j-sell) [inline Assets/Scripts/ToggleFlashlight.cs:20]: This else block is missing brackets.

File: Assets/Scripts/ToggleFlashlight.cs [hunk 2]
```diff
@@ -4,16 +4,22 @@
...
-        light.intensity = 0; //If the light was on, turn it off.
+      	 	light.intensity = 0; //If the light was on, turn it off.
```
Comment (thomas-j-sell) [inline Assets/Scripts/ToggleFlashlight.cs]: Only this line is part of the else.

File: Assets/Scripts/ToggleFlashlight.cs [hunk 3]
```diff
@@ -4,16 +4,22 @@
...
+			lf.brightness = 0;
```
Comment (thomas-j-sell) [inline Assets/Scripts/ToggleFlashlight.cs:23]: This line is out side of the if/else so it always runs.

**Comment by thomas-j-sell** [comment]
Fixing the else block bracketing fixed the issue for me.  I like the idea of the lens flare, but it seems to be a bit much with it always on.  I suggest animating it when the light turns on and letting it fade away, or maybe just subdue it a bit.

**Comment by Haden2** [comment]
Ok, that makes a lot of sense. What happened was when I went to add to the if statement, it told me there was an issue because there was an unexpected else statement. I realized there were no brackets separating the if and else statement but I only added brackets to the if, not the else. That makes sense. 

Animating it will be no issue whatsoever because it is already animated. Change the fade speed from 100 to 1 and you've got it. I had it like that because I was looking at the light as the source instead of a flare per say. Try the welding flare and see if you prefer that one. Fading it away a bit would be a good idea. It is a bit much, but I also wanted a slight handicap when it came to the flashlight. I dont want it to be the best spectrum to see out of. Having the light on increases the chances of the light monster showing up. Cause like the other spectrums have their own handicaps, so I wanted to make the flashlight kind of have a flaw as well, you know? I'll mess around with it some unless you do it before hand.

**Comment by Haden2** [comment]
Ok, this is fixed

**Pull request merged**